app/assets/javascripts/react/components/TouchKiosk.js

```diff
@@ -14,7 +14,8 @@ const mapStateToProps = (state) => {
     api: state.kiosk.api,
     hours: state.touch.hours,
     is_fetching_slides: state.touch.is_fetching_slides,
-    is_fetching_hours: state.touch.is_fetching_hours
+    is_fetching_hours: state.touch.is_fetching_hours,
+    show_nav: true
```

Comment:
add `show_nav` initial state with its default value to the `reducers/touchReducer.js` and then refer to it here as `state.touch.show_nav`.